fetch.bs

```diff
@@ -4903,8 +4951,59 @@ initially a new {{AbortSignal}} object.
  <var>init</var>])</code>
  <dd>Returns a new <var>request</var> whose {{Request/url}} property is <var>input</var> if
  <var>input</var> is a string, and <var>input</var>'s {{Request/url}} if <var>input</var> is a
- {{Request}} object. The optional <var>init</var> argument allows for setting attributes
- appearing in {{RequestInit}} via object members of the same name.
+ {{Request}} object.
+
+ <p>The optional <var>init</var> argument allows for setting properties appearing in {{RequestInit}}
+ via object members of the same name. These are the object members that can be used:</p>
```

Comment:
I think this could be tightened up to not reference the RequestInit dictionary; now that this documentation exists, developers shouldn't need to know about it. E.g. "The optional _init_ argument is an object whose properties can be set as follows:"

Reply:
That sounds good to me